CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v0.52.1
+
+### Breaking changes
+
+- Remove `lock=False` as a default keyword argument to `xr.open_mfdataset` in the `MetDataSource.open_dataset` method. This reverts a change from [v0.44.1](https://github.com/contrailcirrus/pycontrails/releases/tag/v0.44.1) and prevents segmentation faults when using recent versions of [netCDF4](https://pypi.org/project/netCDF4/) (1.7.0 and above).
+- GOES imagery is now loaded from a temporary file on disk rather than directly from memory when using `GOES.get` without a cachestore.
+
+### Internals
+
+- Remove upper limits on netCDF4 and numpy versions.
+- Remove h5netcdf dependency.
+- Update doctests with numpy 2 scalar representation (see [NEP 51](https://numpy.org/neps/nep-0051-scalar-representation.html)).
```

Comment:
So will the doctests now fail with numpy<2?

Reply:
Yes. I added a check in the Makefile (https://github.com/contrailcirrus/pycontrails/pull/213/commits/d39d70e98fea4355fcb7d11191581c33f4d8cbab) to make this requirement explicit. 

Trying to keep doctests compatible with numpy 1 and 2 seems tricky.... You can get the same output for scalars by using `print`, but that doesn't work for scalars wrapped in some other object:
```
>>> np.float64(1.0)
1.0              # numpy 1
np.float64(1.0)  # numpy 2
>>> print(np.float64(1.0)
1.0  # numpy 1
1.0  # numpy 2
>>> print((np.float64(1.0), np.float64(2.0))
(1.0, 2.0)                          # numpy 1
(np.float64(1.0), np.float64(2.0))  # numpy 2
```

Happy to discuss this more if you don't think this seems like the right solution!